src/App.vue

```diff
@@ -238,9 +285,49 @@ export default Vue.extend({
         // Ignore passed value argument.
         this.$store.commit("setUnauthorizedError", null);
       }
+    },
+    availableProjects() {
+      return this.$store.state.projects.projects;
+    },
+    ...mapGetters({
+      project: "projects/getProjectById"
+    }),
+    sensibleColors() {
+      let obj = {};
+      for (let color in colors) {
+        if (["yellow", "shades"].indexOf(color) === -1) {
+          for (let shade in colors[color]) {
+            if (shade.includes("base") || shade.includes("darken2")) {
+              obj[String(color + "_" + shade)] = colors[color][shade];
+            }
+          }
+        }
+      }
+      return obj;
+    }
+  },
+  methods: {
```

Comment:
I was going to suggest to make most of these computed properties but now I think it would be better to move this functionality to the projects store.